app/stac_api/views.py

```diff
@@ -220,8 +222,15 @@ def get_queryset(self):
     def filter_by_bbox(self, queryset, bbox):
         try:
             logger.debug('Item query parameter bbox = %s', bbox)
-            query_bbox_polygon = Polygon.from_bbox(bbox.split(','))
-        except ValueError as error:
+            list_bbox_values = bbox.split(',')
+            if list_bbox_values[0] == list_bbox_values[2] and \
+            list_bbox_values[1] == list_bbox_values[3]:
```

Comment:
I would uses `()` instead of `\` for line breaking in if statement, it makes the code cleaner.